include/cvodes/cvodes.h

```diff
@@ -362,25 +360,25 @@ SUNDIALS_EXPORT void CVodeQuadFree(void* cvode_mem);
 
 /* Initialization functions */
 SUNDIALS_EXPORT int CVodeSensInit(void* cvode_mem, int Ns, int ism,
-                                  CVSensRhsFn fS, N_Vector* yS0);
+                                  CVSensRhsFn fS, N_Vector1d yS0);
 SUNDIALS_EXPORT int CVodeSensInit1(void* cvode_mem, int Ns, int ism,
-                                   CVSensRhs1Fn fS1, N_Vector* yS0);
-SUNDIALS_EXPORT int CVodeSensReInit(void* cvode_mem, int ism, N_Vector* yS0);
+                                   CVSensRhs1Fn fS1, N_Vector1d yS0);
+SUNDIALS_EXPORT int CVodeSensReInit(void* cvode_mem, int ism, N_Vector1d yS0);
 
 /* Tolerance input functions */
 SUNDIALS_EXPORT int CVodeSensSStolerances(void* cvode_mem, sunrealtype reltolS,
-                                          sunrealtype* abstolS);
+                                          sunrealtype1d abstolS);
 SUNDIALS_EXPORT int CVodeSensSVtolerances(void* cvode_mem, sunrealtype reltolS,
-                                          N_Vector* abstolS);
+                                          N_Vector1d abstolS);
 SUNDIALS_EXPORT int CVodeSensEEtolerances(void* cvode_mem);
 
 /* Optional input specification functions */
 SUNDIALS_EXPORT int CVodeSetSensDQMethod(void* cvode_mem, int DQtype,
                                          sunrealtype DQrhomax);
 SUNDIALS_EXPORT int CVodeSetSensErrCon(void* cvode_mem, sunbooleantype errconS);
 SUNDIALS_EXPORT int CVodeSetSensMaxNonlinIters(void* cvode_mem, int maxcorS);
-SUNDIALS_EXPORT int CVodeSetSensParams(void* cvode_mem, sunrealtype* p,
-                                       sunrealtype* pbar, int* plist);
+SUNDIALS_EXPORT int CVodeSetSensParams(void* cvode_mem, sunrealtype1d p,
+                                       sunrealtype1d pbar, int1d plist);
```

Comment:
`doc/shared/sundials/Types.rst` introduced `sunindextype1d` but not `int1d` -- should this be added to the documentation?